.github/workflows/android-build.yml

```diff
@@ -3,7 +3,7 @@ name: android-build
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
There are also references to the master branch in `RELEASE.md` and in the documentation files.

Reply:
Thanks, I missed the references to the master branch in `RELEASE.md`.

However I did not update the documentation files because it concerns the master branch of the [material-components-android](https://github.com/material-components/material-components-android) repository (for instance `https://github.com/material-components/material-components-android/tree/master/lib/java/com/google/android/material/bottomnavigation/BottomNavigationView.java`).